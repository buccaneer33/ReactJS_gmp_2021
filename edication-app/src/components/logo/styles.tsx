import {createUseStyles} from 'react-jss'

export const stylesList = createUseStyles({
    logo: {
        height: '25px',
        color: '#F65261',
        fontSize: '19px',
        padding: '0',
        margin: '0'
    }
});
