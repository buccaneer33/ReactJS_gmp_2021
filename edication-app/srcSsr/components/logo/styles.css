.logo {
    height: 25px;
    color: #F65261;
    font-size: 19px;
    padding: 0;
    margin: 0;
}
