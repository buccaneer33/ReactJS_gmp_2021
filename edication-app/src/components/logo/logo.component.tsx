import React from 'react';
import {stylesList} from './styles';

export const LogoComponent: React.FC = () => {
    const classes = stylesList();
    return (
        <div className={classes.logo}>
            <strong>netflix</strong>roulette
        </div>
    );
}
