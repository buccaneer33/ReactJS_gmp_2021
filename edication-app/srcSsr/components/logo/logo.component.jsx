import React from 'react';
import styles from './styles.css';

export const LogoComponent = () => {
    return (
        <div class={styles.logo}>
            <strong>netflix</strong>roulette
        </div>
    );
}
