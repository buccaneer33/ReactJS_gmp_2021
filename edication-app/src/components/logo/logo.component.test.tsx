import React from 'react';
import {render, screen } from '@testing-library/react';
import {LogoComponent} from './logo.component';

describe('LogoComponent', () => {

    test('true is truthy', () => {
        expect(true).toBe(true);
      });
     
      test('false is falsy', () => {
        expect(false).toBe(false);
      });
  
    test('component conteins "roulette"', () => {
        render(<LogoComponent />);
        expect(screen.getByText('roulette')).toBeInTheDocument();
    });
    test('component conteins "netflix"', () => {
        render(<LogoComponent />);
        expect(screen.getByText('netflix')).toBeInTheDocument();
    });
    test('test snapchot component', () => {
        const component = render(<LogoComponent />).container;
        expect(component).toMatchSnapshot();
    });
});
