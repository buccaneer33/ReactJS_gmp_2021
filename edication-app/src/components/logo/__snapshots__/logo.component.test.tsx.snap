// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoComponent test snapchot component 1`] = `
<div>
  <div
    class="logo-0-2-1"
  >
    <strong>
      netflix
    </strong>
    roulette
  </div>
</div>
`;
